README.md

```diff
@@ -45,6 +45,18 @@ You are all set!
     swe_vis_limit_mag
 ```
 
+## Inline documentation
```

Comment:
I think we should also include an HTML version in this repository. We would update it every time we add new documentation.